pyvelox/pyvelox.h

```diff
@@ -168,4 +257,43 @@ inline void addVeloxBindings(pybind11::module& m, bool asLocalModule = true) {
   rowType.def("names", &RowType::names, "Return the names of the columns");
 }
 
+inline void addVectorBindings(
+    pybind11::module& m,
+    bool asModuleLocalDefinitions = true) {
+  namespace py = pybind11;
+  std::shared_ptr<facebook::velox::memory::MemoryPool> pool =
+      facebook::velox::memory::getDefaultMemoryPool();
+  py::class_<
+      facebook::velox::BaseVector,
```

Comment:
I wonder if this should be flat since we are converting to flat, and it would be nice to add all the methods supported in flat : valueAt, set etc..

Reply:
The thing is `FlatVector` is a templated type, meaning we'd have to explicitly register a `FlatVector<T>` for each `T`. It wouldn't be a problem to do, but this was simpler at the time.